github/src/main/kotlin/gropius/sync/github/utils/TimelineItemHandler.kt

```diff
@@ -72,16 +73,24 @@ class TimelineItemHandler(
     private suspend fun handleIssueClosed(
         imsProjectConfig: IMSProjectConfig, issue: IssueInfo, event: ClosedEventTimelineItemData
     ): Pair<String?, OffsetDateTime?> {
-        var closedEvent = ClosedEvent(event.createdAt, OffsetDateTime.now())
-        closedEvent.issue().value = issue.load(neoOperations)
-        closedEvent.createdBy().value = nodeSourcerer.ensureUser(imsProjectConfig, event.actor!!)
-        closedEvent.lastModifiedBy().value = nodeSourcerer.ensureUser(imsProjectConfig, event.actor!!)
-        closedEvent = neoOperations.save(closedEvent).awaitSingle()
-        return Pair(closedEvent.rawId, event.createdAt)
+        val loadedIssue = issue.load(neoOperations)
```

Comment:
Lines 76 to 88 and 103 to 115 seem to be code duplicates

Reply:
8a0890acf86d1cfc5dae011805e333ded9dee64c

Reply:
Thought so too, but the used fields are not on a base type or interface in the GraphQL generated code from the github api, meaning refactoring it together would mean passing each used field separately BECAUSE GITHUB IS A FUCKING PIECE OF GARBAGE!!!!!!

Reply:
Okay, understood 😿